src/mpi/errhan/errnames.txt

```diff
@@ -899,6 +899,8 @@ is too big (> MPIU_SHMW_GHND_SZ)
 **mpir_wingetattr:MPII_Win_get_attr failed
 **mpir_wingetattr %W %d %p %p:MPII_Win_get_attr(%W, win_keyval=%d, attribute_val=%p, flag=%p) failed
 
+## QMPI error messages
+**qmpi_get_function_table: QMPI_Get_function_table failed
```

Comment:
Where is this error name used? 

I ask because the MPIR_Err routines may not work outside MPI (between `Init/Finalize`). Can we simply use a MPI_T error code, or use an `MPIX` error code?

Reply:
This was a holdover from a previous version of the code. It's not needed now.